### PR TITLE
RELATED: RAIL-3492 - Change KPI Measure handler + fixture refactors

### PR DESCRIPTION
**Pull request by lupko**
---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4795/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6435/ : FAILURE in 6m 23s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4797/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6437/ : FAILURE in 7m 22s

File: libs/sdk-ui-dashboard/src/model/tests/fixtures/Insights.fixtures.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export const PivotTableWithDateFilter = recordedInsight(
+    ReferenceRecordings.Insights.PivotTable.WithDateFilter,
+);
+export const PivotTableWithTwoSameDates = recordedInsight(
+    ReferenceRecordings.Insights.PivotTable.WithTwoSameDates,
+);
+export const TreemapWithSingleMeasureAndViewByFilteredToOneElement = recordedInsight(
+    ReferenceRecordings.Insights.Treemap.SingleMeasureAndViewByFilteredToOneElement,
+);
+// (C) 2021 GoodData Corporation
```
Comment (no23reason) [inline libs/sdk-ui-dashboard/src/model/tests/fixtures/Insights.fixtures.ts:17]: Detail: extra copyright

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4799/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6439/ : FAILURE in 7m 23s

File: libs/sdk-ui-dashboard/src/model/commands/kpi.ts [hunk 2]
```diff
@@ -73,8 +73,11 @@ export interface ChangeKpiWidgetMeasure extends IDashboardCommand {
         /**
          * Optionally specify the new header that should be used for the KPI widget with the
          * changed measure.
+         *
+         * You may specify the widget header as 'from-measure'. In that case the title will be automatically
```
Comment (no23reason) [inline libs/sdk-ui-dashboard/src/model/commands/kpi.ts:77]: The comment mentions "from-measure" but the type has "from-title". This should be in sync, right?

Comment (lupko) [reply libs/sdk-ui-dashboard/src/model/commands/kpi.ts:77]: good catch; fixed, renamed to 'from-measure'

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: libs/sdk-ui-dashboard/src/model/tests/fixtures/Insights.fixtures.ts (hunk 1 above)
Comment (lupko) [reply libs/sdk-ui-dashboard/src/model/tests/fixtures/Insights.fixtures.ts]: good catch; IntelliJ refactoring was smart again :)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4802/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6441/ : SUCCESS in 23m 57s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1359/ : SUCCESS in 1m 17s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1635/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/419/ : SUCCESS in 12m 48s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6443/ : SUCCESS in 16m 44s

**Pull request merged**